scripts/cleanup-tier-resources.sh

```
@@ -0,0 +1,219 @@
+elif [[ "$local_models_output" == *"error"* ]] || [[ "$local_models_output" == *"Error"* ]]; then
+    log_error "Failed to list LLMInferenceServices in $MODEL_NAMESPACE: $(echo "$local_models_output" | grep '^error:' | tail -1)"
+    cleanup_failed=true
+else
+    models="$local_models_output"
+fi
+
+if [[ -n "$models" ]]; then
+    while IFS= read -r model; do
+        [[ -z "$model" ]] && continue
+        if [[ "$DRY_RUN" == "true" ]]; then
+            local has_annotation
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)